test/unit/util/__init__.py

```diff
@@ -17,5 +23,34 @@ def test_dict_merge(self):
         dict3 = {'a': {'b': {'c': 1, 'd': 1}}, 'c': 2, 'd': 2}
         self.assertEqual(hca.util._merge_dict(dict1, dict2), dict3)
 
+
+class TestLinking(TmpDirTestCase):
+    """TmpDirTestCase will ensure any links / files we create are cleaned up"""
```

Comment:
I don't like one-liner docstrings because I like the vertical space introduced by the two lines with just triple quotes on them. But there is no clear guide for this repo so I'll leave this to you.

Reply:
Since this format of docstring is still PEP 8, I think it is acceptable to leave it as is.